src/lib/db/migrations/20230310111029_initial.ts

```diff
@@ -18,18 +18,21 @@ export async function up(knex: Knex): Promise<void> {
     def.primary(['id'])
   })
 
-  await knex.schema.createTable('example', (def) => {
+  await knex.schema.createTable('certificate', (def) => {
     def.uuid('id').defaultTo(knex.raw('uuid_generate_v4()'))
+    def.integer('capacity').notNullable().index('capacity_index')
+    def.integer('co2e').notNullable().index('co2e_index')
     def.string('owner', 48).notNullable()
     def
-      .enum('state', ['created', 'allocated'], {
-        enumName: 'example_state',
+      .enum('state', ['initialized', 'issued', 'revoked'], {
+        enumName: 'certificate_state',
         useNative: true,
       })
       .notNullable()
-    def.uuid('parameters_attachment_id').notNullable()
-    def.integer('latest_token_id')
-    def.integer('original_token_id')
+      .defaultTo('initialized')
+    def.uuid('parameters_attachment_id').nullable().defaultTo(null)
```

Comment:
do we have a `parameters_attachment_id` anywhere else? What does it contain

Reply:
Mmm. apart from where you have hightlighted below, which has been removed.